src/ReactWrapper.js

```diff
@@ -62,7 +62,7 @@ export default class ReactWrapper {
     if (!global.window && !global.document) {
       throw new Error(
         'It looks like you called `mount()` without a jsdom document being loaded. ' +
-        'Make sure to only use `mount()` inside of a `describeWithDOM(...)` call. '
+        'Make sure to only use `mount()` after initialising jsdom (see http://airbnb.io/enzyme/docs/guides/jsdom.html)'
```

Comment:
i agree with @lelandrichardson here - enzyme works in browsers too, so it's not about jsdom as it is about having a global `document`.